scss/_variables.scss

```diff
@@ -98,8 +98,8 @@ $yiq-text-light: $white !default;
 
 $enable-caret:              true !default;
 $enable-rounded:            true !default;
-$enable-shadows:            false !default;
-$enable-gradients:          false !default;
+$enable-shadows:            true !default;
```

Comment:
Do we want to leave shadows and gradients on by default?

Reply:
Nope! Not yet at least :).